tenable/io/v3/vm/exclusions/api.py

```diff
@@ -0,0 +1,443 @@
+        Args:
+            fields (list, optional):
+                The list of field names to return from the Tenable API.
+                Example:
+                    - ``['field1', 'field2']``
+            filter (tuple, Dict, optional):
+                A nestable filter object detailing how to filter the results
+                down to the desired subset.
+                Examples:
+                    >>> ('or', ('and', ('test', 'oper', '1'),
+                                   ('test', 'oper', '2')
+                            ),
+                        'and', ('test', 'oper', 3))
+                    >>> {'or': [
+                    {'and': [
+                        {'value': '1', 'operator': 'oper', 'property': '1'},
+                        {'value': '2', 'operator': 'oper', 'property': '2'}
+                        ]
+                    }],
+                    'and': [
+                        {'value': '3', 'operator': 'oper', 'property': 3}
+                        ]
+                    }
+                As the filters may change and sortable fields may change over
+                time, it's highly recommended that you look at the output of
+                the :py:meth:`tio.v3.vm.filters.audit_log_filters()`
+                endpoint to get more details.
+            sort (list[tuple], optional):
+                A list of dictionaries describing how to sort the data
+                that is to be returned.
+                Examples:
+                    - ``[('field_name_1', 'asc'), ('field_name_2', 'desc')]``
+            limit (int, optional):
+                Number of objects to be returned in each request.
+                Default and max_limit is 200.
+            next (str, optional):
+                The pagination token to use when requesting the next page of
+                results. This token is presented in the previous response.
+            return_resp (bool, optional):
+                If set to true, will override the default behavior to return
+                an iterable and will instead return the results for the
+                specific page of data.
+            return_csv (bool, optional):
+                If set to true, it will return the CSV response or
+                iterable (based on return_resp flag). Iterator returns all
+                rows in text/csv format for each call with row headers.
+        Returns:
+            Iterable:
+                The iterable that handles the pagination for the job.
+            requests.Response:
+                If ``return_json`` was set to ``True``, then a response
+                object is instead returned instead of an iterable.
+        Examples:
+            >>> tio.v3.vm.exclusions.search(
+            ...     fields=['id', 'name'],
+            ...     filter=('id', 'eq', [1,2]),
+            ...     sort=[('id', 'asc')],
+            ...     limit=10
+            ... )
+        '''
+        iclass = ExclusionSearchIterator
```

Comment:
Aren't we using SearchIterator and CSVChunkIterator?

Reply:
updated.